graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/execution/ApolloSubscriptionProtocolHandler.kt

```diff
@@ -107,6 +107,11 @@ class ApolloSubscriptionProtocolHandler(
             return Flux.just(basicConnectionErrorMessage)
         }
 
+        if (sessionState.operationExists(session, operationMessage)) {
```

Comment:
I was thinking about this earlier and I don't think we need to have this check. There is only every one session open per connection so we are not going to every have a client open a connection with duplicate session IDs and request ID.

If that some how did happen I imagine we would want to return the active subscription but I don't think we need to handle this

Reply:
We will overwrite the value in the map, so the privious one won't be closed, and you might get messages double.

Reply:
Another way to handle would be when we add the subscription, to first cancel any existing one.